Formula/qgis3.rb

```diff
@@ -809,53 +865,33 @@ def brewed_grass7?
   # end
 
   def brewed_python?
-    Formula["python3"].linked_keg.exist?
+    Formula["python"].linked_keg.exist?
   end
 
   def python_exec
     if brewed_python?
-      Formula["python3"].opt_bin/"python3"
+      Formula["python"].opt_bin/"python3"
```

Comment:
If we're prepending the vendored python to the path, shouldn't we set that as the python executable for the rest of the formula to use?

Reply:
You say use: `libexec/"python/bin/pyton3"`

Reply:
That would seem to be the correct approach. If we're relying on the vendored python packages, we should use the vendored python installation as well.

Reply:
Working on minor changes, later I upload everything.
And I will test what happens with the GRASS scripts comparing building with `python 3`.

Reply:
Ok. Thanks!

Reply:
When building GRASS with python3 please make sure we don't remove support for python2 scripts.

Reply:
The changes I made in the GRASS formula

```
# ensure python3 is used
bin.env_script_all_files(libexec/"bin", :GRASS_PYTHON => "python3")
```
```
def formula_site_packages(f)
  `python3 -c "import os, sys, site; sp1 = list(sys.path); site.addsitedir('#{Formula[f].opt_lib}/python3.7/site-packages'); print(os.pathsep.join([x for x in sys.path if x not in sp1]))"`.strip
end
```

Reply:
When you're ready, open a PR and we can go through it. Trying to keep these ticket treads from getting too long.

Reply:
Ok, let's close this PR.